nece/managers.py

```diff
@@ -96,8 +96,11 @@ def language(self, language_code):
 
     def _clone(self):
         """Override `_clone` method in order to inject the `language_code`."""
+        language_code = self._language_code
+        if language_code is None:
+            language_code = get_language().replace("-", "_")
         clone = super()._clone()
-        clone._language_code = self._language_code
+        clone._language_code = language_code
```

Comment:
We can even more simplify this:
`clone._language_code = self._language_code or get_language().replace("-", "_")`